### PR TITLE
Added a note on how to contribute in the README

### PR DESCRIPTION
**Pull request by phillip-kruger**
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

As discussed in the Community meeting, making "How to contribute" more visible

### CODE REVIEW

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by phillip-kruger** [comment]
As discussed here: https://groups.google.com/forum/#!topic/microprofile/-o0Mqv4I2IY

Moved the "How to contribute" to it's own file

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
```
Comment (arthurdm) [inline CONTRIBUTING.adoc]: very minor, but probably should be 2019?

Comment (phillip-kruger) [reply CONTRIBUTING.adoc]: Changed to 2019

**Comment by arthurdm** [comment]
thanks @phillip-kruger

**Pull request merged**